db-client/src/main/java/com/splicemachine/db/client/net/NetConnection40.java

```diff
@@ -449,4 +458,15 @@ public void setNetworkTimeout( Executor executor, int milliseconds ) throws SQLE
         throw SQLExceptionFactory.notImplemented ("setNetworkTimeout");
     }
 
+    private Object sideConnectionLock = new Object();
+    public NetConnection getSideConnection() throws SqlException {
+        synchronized (sideConnectionLock) {
+            if (sideConnection_ == null || !sideConnection_.isOpen())
+                sideConnection_ = new NetConnection40(null, 0,
```

Comment:
What does `null` represent? Could we make it clearer by assigning a name? How about `0`?